### ISSUE TITLE
STU3 version

### ISSUE THREAD

**Issue by Abildtoft** [fhirbase/fhirbase-plv8#175]
Hi,

Thanks for an awesome tool!

Are there any plans to update FHIRBase to FHIR 3.0.0 (STU3) now that 3.0.0 has been released?

**Comment by niquola** [comment]
I will try to allocate some time to do it!

**Comment by merqurio** [comment]
How can we help with this ? We can put some effort on this with some guidance.

**Comment by niquola** [comment]
Yep, try to download stu3 metadata into https://github.com/fhirbase/fhirbase-plv8/tree/master/fhir, then run build and tests - report what does not work
